client/src/entrypoints/contrib/table_block/table.js

```diff
@@ -173,15 +170,11 @@ function initTable(id, tableOptions) {
   });
 
   hot = new Handsontable(document.getElementById(containerId), finalOptions);
-  hot.render(); // Call to render removes 'null' literals from empty cells
-
-  // Apply resize after document is finished loading (parent .sequence-member-inner width is set)
-  if ('resize' in $(window)) {
+  window.addEventListener('load', () => {
+    // Render the table. Calling render also removes 'null' literals from empty cells.
+    hot.render();
     resizeHeight(getHeight());
-    $(window).on('load', () => {
-      $(window).trigger('resize');
-    });
-  }
+  });
```

Comment:
```suggestion
    resizeWidth(getWidth());
  });
```

This also needed resize width on load, the width was not correct if the viewport was less than ~900px.

I will add this change, the rest looks good - thanks @zerolab !